war/src/main/less/base/style.less

```
@@ -100,11 +100,10 @@ td.center-align {
     color: white;
     background-color: #ef2929;
     text-align: center;
-    margin-left: 2em;
-    margin-right: 2em;
-    margin-bottom: 0.5em;
+    margin-bottom: 1.5em;
```

```suggestion
    margin-bottom: var(--section-padding);
```

Would you mind changing this? Haven't tested admittedly but it should increase the bottom margin ever so slightly just to bring it inline with other controls.

Looks good to me though! 🥳

---

Good catch